libraryversions.toml

```diff
@@ -89,7 +89,7 @@ LEANBACK_PREFERENCE = "1.2.0-beta01"
 LEANBACK_TAB = "1.1.0-beta01"
 LEGACY = "1.1.0-alpha01"
 LIBYUV = "0.1.0-dev01"
-LIFECYCLE = "2.9.0"
+LIFECYCLE = "2.9.1"
```

Comment:
> artifactRedirection.version.androidx.lifecycle=2.9.2

but here it's 2.9.1. Is it expected?

Reply:
Nope. Good catch.
Actually we don't use the value from this place, so it won't break anything.

Let me update it anyway for consistency